.vscode/settings.json

```diff
@@ -9,5 +9,7 @@
         "rulePaths": ["../scripts/eslint/built/rules/"],
         "ext": [".ts"]
     },
-    "eslint.workingDirectories": ["./src", "./scripts"]
+    "eslint.workingDirectories": ["./src", "./scripts"],
+    "typescript.tsserver.log": "normal",
```

Comment:
???

Reply:
Gah sorryyyy

Reply:
It's alright, this bad boy only got committed to the repo with the eslint migration, and this problem is _exactly_ why we didn't commit it before - there's no concept of "per-user workspace settings", so this kinda falls flat. @mjbvz https://github.com/microsoft/vscode/issues/1435 or https://github.com/microsoft/vscode/issues/15909 when - being able to have a `settings.json` in the `.vscode` folder and a `settings.user.json` alongside it would be very nice.

Reply:
Ok, I thought it had been ignored before. I always keep my tsdk the local one so I can test my changes immediately.